.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
We should mention a few things related to PR titles:

Good:
- Prefix the title with the codeline E.g. `4.x: foo bar`
- Shorter titles are better (listing)

Bad:
- Fix issue 1234
- Backport issue 1234

Reply:
Good ideas but see previous comment.